src/bindings/js/node/src/core_wrap.cpp

```diff
@@ -230,3 +233,46 @@ Napi::Value CoreWrap::get_available_devices(const Napi::CallbackInfo& info) {
 
     return js_devices;
 }
+
+Napi::Value CoreWrap::set_property(const Napi::CallbackInfo& info) {
+    try {
+        CoreSetPropertyArgs* args;
+        args = new CoreSetPropertyArgs(info);
+
+        if (args->device_name.empty()) {
```

Comment:
Here is also a check whether `device_name` is specified

Reply:
Any proposals?